network-api/networkapi/templates/donate/fragments/help_page_notice.html

```diff
@@ -1,18 +1,17 @@
 {% load static i18n l10n wagtailcore_tags wagtailimages_tags %}
 
 <div>
-    <div class="tw-p-4 medium:tw-p-8 tw-my-24 tw-w-full tw-flex tw-flex-col small:tw-flex-row tw-items-center tw-border-t tw-border-b tw-border-black">
-        <div class="tw-flex-shrink-0 tw-grow-0 tw-basis-[50px] tw-mr-4 medium:tw-mr-8">
-
-            <img src="{% static "_images/donate/megaphone.png" %}" alt="icon" class="tw-w-full tw-h-full" width="100" height="100">
+    <div class="tw-py-4 tw-px-0 large:tw-p-8 tw-my-24 tw-w-full tw-flex tw-flex-col large:tw-flex-row tw-items-start large:tw-items-center tw-border-t tw-border-b tw-border-black">
+        <div class="tw-flex-shrink-0 tw-grow-0 tw-basis-[50px] tw-mr-4 large:tw-mr-8">
```

Comment:
We should avoid using custom Tailwind units, like the `tw-basis-[50px]`. We should rewrite this as `tw-basis-24`, as 24 units in our config is 3rem (50px).

Reply:
Thanks @jhonatan-lopes! Just updated 👍